Makefile

```diff
@@ -31,6 +31,9 @@ deps:
         (go get github.com/laher/goxc)
 	@which goimports > /dev/null || \
         (go get golang.org/x/tools/cmd/goimports)
+	@which porter1 > /dev/null || \
```

Comment:
What is `porter1` and why do we need it?

Reply:
Fix